addons/l10n_mx/data/account_configuration.yml

```diff
@@ -0,0 +1,10 @@
+-
+  Account settings set for Use Anglo-Saxon Accounting
+-
+  !record {model: account.config.settings, id: mx_default_account_properties}:
+    {
+      'use_anglo_saxon': 1,
```

Comment:
No, your module shouldn't magically auto-configure what you don't want to configure in all of your instances. If you do that, other localization module can do the reverse. Then, if you install both, your system will be again "unconfigured".